searx/engines/docker_hub.py

```diff
@@ -19,14 +19,17 @@
 categories = ['it', 'packages']  # optional
 paging = True
 
-base_url = "https://hub.docker.com/"
-search_url = base_url + "api/content/v1/products/search?{query}&type=image&page_size=25"
+base_url = "https://hub.docker.com"
+page_size = 10
```

Comment:
removing the slash here, needs to add the slash there ..

```diff
diff --git a/searx/engines/docker_hub.py b/searx/engines/docker_hub.py
index 1aa25e39f..021c05827 100644
--- a/searx/engines/docker_hub.py
+++ b/searx/engines/docker_hub.py
@@ -56,7 +56,7 @@ def response(resp):
 
         result = {
             'template': 'packages.html',
-            'url': base_url + ("_/" if is_official else "r/") + item.get("slug", ""),
+            'url': base_url + ("/_/" if is_official else "/r/") + item.get("slug", ""),
             'title': item.get("name"),
             'content': item.get("short_description"),
             'thumbnail': item["logo_url"].get("large") or item["logo_url"].get("small"),
```

Reply:
True, I changed the `base_url` at the end of my changes but forgot to change the links, good catch :+1: